doc/examples/applications/plot_cornea_spot_inpainting.py

```diff
@@ -0,0 +1,91 @@
+"""
+============================================================
+Reconstruct dust-covered human cornea image using inpainting
+============================================================
+
+Optical Coherence Tomography (OCT) [1]_ is used to provide eye doctors with an
+image of the retina in the back of a patient's eye.
+
+Quite commonly, there is dust on the reference mirror of the equipment,
+causing dark spots to appear on the acquired images.
+
+This tutorial is adapted from an application shared by Jules Scholler in [2]_.
+The images were acquired by Viacheslav Mazlin.
+
+.. [1] Vinay A. Shah, MD (2015)
+       `Optical Coherence Tomography <https://eyewiki.aao.org/Optical_Coherence_Tomography#:~:text=3%20Limitations-,Overview,at%20least%2010%2D15%20microns.>`_,
+       American Academy of Ophthalmology.
+.. [2] Jules Scholler (2019) "Image denoising using inpainting"
+       https://www.jscholler.com/2019-02-28-remove-dots/
+
+"""
+
+import imageio.v3 as iio
+import numpy as np
+import plotly.io
+import plotly.express as px
+
```

Comment:
```suggestion
import matplotlib.pyplot as plt
```

Reply:
Hey @mkcor ! I noticed that we haven't imported `matplotlib` here. Are we not supposed to use the `pyplot` package and use `plotly` instead?

Reply:
If you build the docs locally and/or use the pre-commit hook, you will notice that you can't have unused imports. So, basically, only import `matplotlib` if/when you are going to use it -- the same goes with committing this addition :wink:

Reply:
Also, I recommend importing submodules rather than functions (cf. https://github.com/scikit-image/scikit-image/pull/5365#issuecomment-855788844).

Reply:
Please sort the imports in alphabetical order.

Reply:
@ana42742 please hold on: Refrain from making changes locally; I'll push a few updates; then I'll let you sync up your local branch. Thanks!

Reply:
Okay @mkcor!

Reply:
Thanks, @ana42742! You are welcome to resume the regular contribution workflow now. Cc'ing @decorouz.